app/connectors/Acc40Connector.scala

```diff
@@ -75,7 +75,7 @@ class Acc40Connector @Inject()(httpClient: HttpClient,
 
   def searchType(searchID: EORI) = {
     searchID.value match {
-      case x if x.startsWith("GB") => "0"
+      case x if x.startsWith("GB") || x.startsWith("XI") => "0"
```

Comment:
Better to rename the x to some meaningful name like id etc

Reply:
Renamed it to searchEori